README.md

```diff
@@ -41,6 +41,25 @@ For an even easier usage, just `#import <KVOController/NSObject+FBKVOController.
 [self.KVOController observe:clock keyPath:@"date" options:NSKeyValueObservingOptionInitial|NSKeyValueObservingOptionNew action:@selector(updateClockWithDateChange:)];
 ```
 
+## Swift
+
+KVOController works fine with Swift but the properties that you need to observer must be marked as `dynamic` check the following example:
+```Swift
+class TasksListViewModel: NSObject, TasksListViewModelProtocol{
```

Comment:
For me I see KVOController is more useful while working with MVVM to bind properties between the view model and the view. That's why I mentioned an example uses view model. The view deals only with a protocol which can be implemented with any view model. 
The space before opening bracket is fixed  :)